python/tvm/relay/frontend/tflite.py

```diff
@@ -109,16 +109,17 @@ def __init__(self, model, subgraph, exp_tab):
             'PAD': self.convert_pad,
             'POW': self.convert_pow,
             'PRELU': self.convert_prelu,
-            'REDUCE_ANY': self._convert_reduce_any,
-            'REDUCE_MAX': self._convert_reduce_max,
-            'REDUCE_MIN': self._convert_reduce_min,
-            'REDUCE_PROD': self._convert_reduce_prod,
+            'REDUCE_ANY': self.convert_reduce_any,
+            'REDUCE_MAX': self.convert_reduce_max,
+            'REDUCE_MIN': self.convert_reduce_min,
+            'REDUCE_PROD': self.convert_reduce_prod,
             'RELU':self.convert_relu,
```

Comment:
Why are these changes relevant to this Pull request ?

Reply:
@u99127 for code consistency i changed, its not related to this PR. 
This 5 methods were starting with `_` and i removed those.

Reply:
In an ideal world that would be a separate PR rather than merging it in here as that could just go in separately and pretty mechanically.

However, that's really not our development practice yet. 

Ramana

Reply:
@u99127 Thanks for the review. I totally agree with you. I removed those changes as part of this PR. i will raise another PR[#5515] for those. Could you please check again. TIA.